facade/disabledStatusMetrics.go

```diff
@@ -0,0 +1,54 @@
+package facade
```

Comment:
please move this in /facade/disabled package or add unit tests.

Reply:
ok. moved also the `disabledNodeFacade`